ext/iconv/tests/iconv_strpos.phpt

```diff
@@ -41,7 +41,7 @@ bool(false)
 bool(false)
 int(5)
 int(5)
-Offset not contained in string
+strpos(): Argument #3 ($offset) must be within argument #1 ($haystack)
```

Comment:
I'm not 100% sure about the new wording. Maybe "must be within the length ..." or something similar would be more correct?

Reply:
There's an existing use of `must be contained in argument #1 ($haystack)` in `substr_count`. I think the general message is fine, but should use the same word contained/within. Mild preference towards contained.

Reply:
somehow, the "must be contained" feels a bit artificial for me. @Girgias do you maybe have any suggestion? But in any case, I'll make sure that these messages use the same words as `substr_count()`

Reply:
Other than using the word ``bound`` (as in Out of bounds) I don't really have any good idea e.g. ``... offset must be in the bound of haystack`` but that looks awkward.
I don't see the issue with contained/within but I'd agree with Nikita that contained does seem to sound slightly more natural.

Reply:
Thanks! Then let's go with the `contained` stuff.

Reply:
It's fixed in my latest commit (along with the other comments). Unfortunately, I forgot not to amend.